examples/nanoChatGPT/train_rlhf.py

```diff
@@ -71,41 +99,98 @@ def forward(self, tensordict):
     max_iters = config["max_iters"]
     num_epochs = config["num_epochs"]
     device = config['device']
+    num_envs = config["batch_size"]
+    grad_clip = config["grad_clip"]
+
+    total_frames = max_iters * num_envs * ep_length
     rb = TensorDictReplayBuffer(
-        storage=LazyTensorStorage(ep_length * config["batch_size"]),
+        storage=LazyTensorStorage(ep_length * num_envs),
         batch_size=config["ppo_batch_size"],
         sampler=SamplerWithoutReplacement(),
     )
     rewards = []
     losses = []
+    
+    # a quick rollout to init the actor
+    with torch.no_grad():
+        td = env.rollout(3, actor)
+    del td
+
+    test_config = deepcopy(config)
+    test_config["batch_size"] = 1
+    test_config["episode_length"] = 50
+    train_loader_test, _ = get_dataloaders(test_config)
+    test_env = RLHFEnv(reward_model=reward_model, config=test_config, dataloader=train_loader_test)
+
+    def test():
+        training = actor.training
+        actor.eval()
+        with set_exploration_type(ExplorationType.MODE), torch.no_grad():
```

Comment:
we explicitly say we don't want to sample actions but take the most probable one